code/modules/client/preferences.dm

```diff
@@ -244,6 +244,16 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 				dat += "<span style='border: 1px solid #161616; background-color: #[active_character.features["ethcolor"]];'>&nbsp;&nbsp;&nbsp;</span> <a href='?_src_=prefs;preference=color_ethereal;task=input'>Change</a><BR>"
 
+			if("apidcolor" in active_character.pref_species.default_features)
+
+				if(!use_skintones)
+					dat += APPEARANCE_CATEGORY_COLUMN
+
+				dat += "<h3>Apid Color</h3>"
+
+				dat += "<span style='border: 1px solid #161616; background-color: #[active_character.features["apidcolor"]];'>&nbsp;&nbsp;&nbsp;</span> <a href='?_src_=prefs;preference=color_apid;task=input'>Change</a><BR>"
+
+
```

Comment:
I am assuming this is the codespaghetti attempt at making unique Apid colors. In which case, this could be removed. I don't see `apidcolor` used anywhere in the species definition.

Reply:
I think kitz just erroneously copied the ethcolor above, but considering the way apid colors apply with mutantcolor, it should be fine.